py/selenium/webdriver/chromium/options.py

```diff
@@ -51,6 +51,8 @@ def binary_location(self, value: str) -> None:
         :Args:
          - value: path to the Chromium binary
         """
+        if not isinstance(value, str):
+            raise TypeError("Binary Location must be a string")
```

Comment:
Since we're repeating the message all over, it might be worth added the error messages to variables and use that? Since we have the base class the error messages could live there and then we know each subclass has the same error message.

```suggestion
            raise TypeError(self.BINARY_LOCATION_ERROR)
```

Reply:
Yes. We can include all common error messages in base class so that we can make use of it in sub-classes. let me give a try.

Reply:
> Since we're repeating the message all over, it might be worth added the error messages to variables and use that? Since we have the base class the error messages could live there and then we know each subclass has the same error message.

Added class variable `BINARY_LOCATION_ERROR` in `ArgOptions` class to hold common binary error message.